### ISSUE TITLE
Server not detected

### ISSUE THREAD

**Issue by odevroed** [andrewnguonly/Lumos#116]
Hi, 

the pop-up sais 'unable to connect to ollama api'. However, I have the correct address and another UI is correctly accessing the API.

When  I do 'inspect pop-up', I get the following error:
Access to fetch at 'http://127.0.0.1:11434/api/tags' from origin 'chrome-extension://lchglknfjpkcpceecjmopnlgekdlogae' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

Any idea what might go wrong?

**Comment by andrewnguonly** [comment]
The Ollama server needs to be started with the `OLLAMA_ORIGINS` environment variable set to `chrome-extension://*`.

Example:
```
OLLAMA_ORIGINS=chrome-extension://* ollama serve
```
I'll update the [documentation](https://github.com/andrewnguonly/Lumos?tab=readme-ov-file#start-server) to be more explicit about the error.

**Comment by fatinghenji** [comment]
I'm having the same problem, but I'm using the edge browser, how should I set the environment variables?
![image](https://github.com/andrewnguonly/Lumos/assets/37131177/cb1eca8f-b119-4952-aff2-2c8bfcb964eb)
Is that what it looks like?

**Comment by andrewnguonly** [comment]
@fatinghenji, for Microsoft Edge browser, the Ollama server should be started with `OLLAMA_ORIGINS=chrome-extension://*`. I typically start the Ollama server from the command line (`OLLAMA_ORIGINS=chrome-extension://* ollama serve`). I'm not familiar with the screenshot you shared.

**Comment by odevroed** [comment]
@andrewnguonly this indeed worked like a charm. Maybe you can add that for Linux, one should add this to the config of systemctl. The way to do this is to edit /etc/systemd/system/ollama.service and add
   Environment="OLLAMA_ORIGINS=chrome-extension://*"
before the execstart statement.

Thanks a lot for your help.

**Comment by andrewnguonly** [comment]
> Maybe you can add that for Linux, one should add this to the config of systemctl. The way to do this is to edit /etc/systemd/system/ollama.service and add
Environment="OLLAMA_ORIGINS=chrome-extension://*"
before the execstart statement.

I'll find a place to document this :+1:

**Comment by andrewnguonly** [comment]
@fatinghenji, I'm closing this issue for now, but feel free to reopen it or create a new issue if you're still having trouble connecting on Microsoft Edge. Instructions for installing on Microsoft Edge are [here](https://github.com/andrewnguonly/Lumos/blob/1.0.10/docs/edge.md).
